src/aui-form-builder/js/aui-form-builder-field-list.js

```diff
@@ -329,6 +329,12 @@ A.FormBuilderFieldList  = A.Base.create('form-builder-field-list', A.Base, [], {
             writeOnce: 'initOnly'
         },
 
+        /**
+         * Attribute responsible for enable or disable add fields.
```

Comment:
**enable** -> **enabling**
**disable** -> **disabling**

`Attribute responsible for enabling or disabling the feature of adding fields`